### PR TITLE
Update getting-started.md - fix MacOS Packaging instructions

### PR DESCRIPTION
**Pull request by rhusar**

### CODE REVIEW

File: content/en/docs/collector/getting-started.md [hunk 1]
```diff
@@ -225,17 +225,16 @@ sudo journalctl -u otelcol
...
+{{< ot-tab lang="console">}}
+$ curl -O -L https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol*{{% param collectorVersion %}}\_darwin_amd64.tar.gz
+$ tar -xvf otelcol*{{% param collectorVersion %}}\_darwin_amd64.tar.gz
+{{< /ot-tab >}}
+
+{{< ot-tab lang="console">}} 
+$ curl -O -L https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol*{{% param collectorVersion %}}\_darwin_arm64.tar.gz
+$ tar -xvf otelcol*{{% param collectorVersion %}}\_darwin_arm64.tar.gz
+{{< /ot-tab >}}
 {{< /ot-tabs >}}
```
Comment (chalin) [inline content/en/docs/collector/getting-started.md:238]:
```suggestion
<!-- prettier-ignore-start -->
{{< ot-tabs Intel ARM >}}
{{< ot-tab lang="console">}}
$ curl -O -L https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol*{{% param collectorVersion %}}\_darwin_amd64.tar.gz
$ tar -xvf otelcol*{{% param collectorVersion %}}\_darwin_amd64.tar.gz
{{< /ot-tab >}}

{{< ot-tab lang="console">}} 
$ curl -O -L https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol*{{% param collectorVersion %}}\_darwin_arm64.tar.gz
$ tar -xvf otelcol*{{% param collectorVersion %}}\_darwin_arm64.tar.gz
{{< /ot-tab >}}
{{< /ot-tabs >}}
<!-- prettier-ignore-end -->
```

**Review by chalin — COMMENTED** [review]
Oh! Thanks for spotting that. You'll need to add some Prettier ignore directives. See inline comment. After accepting the suggested change, the checks should pass.

**Comment by rhusar** [comment]
Fun stuff, thanks @chalin.

**Review by chalin — APPROVED** [review]
👍

**Pull request merged**